pages/smart-contracts-development/development.tsx

```diff
@@ -1,29 +1,29 @@
-'use client';
+"use client";
 
-import Image from 'next/image';
-import PageContainer from '../../components/PageContainer/PageWrapper';
-import { SERVICES_CONTENT_INF0 } from '../../constants/textConstants';
-import AuditCardItem from '../smart-contract-audits/auditsCardItem';
-import styles from '../smart-contract-audits/styles.module.css';
+import Image from "next/image";
+import PageContainer from "../../components/PageContainer/PageWrapper";
+import { SERVICES_CONTENT_INF0 } from "../../constants/textConstants";
+import AuditCardItem from "../smart-contract-audits/auditsCardItem";
+import styles from "../smart-contract-audits/styles.module.css";
 
 const DevelopContract = () => {
   const DEVELOPMENT_CONTENT_ARRAY =
-    SERVICES_CONTENT_INF0['Smart Contracts Development'];
+    SERVICES_CONTENT_INF0["Smart Contracts Development"];
   return (
-    <PageContainer title={'Smart Contracts Development'} link={'/#Services'}>
+    <PageContainer title={"Smart Contracts Development"} link={"/#Services"} isArrow={true}>

```

Comment:
You use in all PageContainer - isArrow={true}. In one situation false. Maybe you can create default true, when you need false - you change this